lib/negroku/tasks/eye.rake

```diff
@@ -69,6 +83,23 @@ namespace :negroku do
       end
     end
 
+    before "eye:setup", "eye:setup:discovery" do 
+      if was_required? 'capistrano3/unicorn'
+            watch_process(:unicorn, fetch(:unicorn_roles), {
```

Comment:
I thought that the only task responsible for monitoring should be eye. 

"eye" wouldnt like other tasks to mess around with its way of monitoring. Also it would be easier to decouple eye on the future.